### PR TITLE
[38] Change colors in theme

### PR DESCRIPTION
**Pull request by bobbylej**
- Change color to monochromatic
- Use dark or light mode based on the browser preferences

### CODE REVIEW

**Review by lukaszzadamczyk — APPROVED** [review]

File: src/app/shared/constants/theme.ts [hunk 1]
```diff
@@ -56,23 +56,23 @@ export const darkTheme = createMuiTheme({
...
+      dark: '#45B39D',
       contrastText: '#fff',
     },
     secondary: {
-      main: '#f50057',
-      light: 'rgb(247, 51, 120)',
-      dark: 'rgb(171, 0, 60)',
+      main: '#33a68f',
+      light: '#3cc7ab',
+      dark: '#237a69',
```
Comment (lukaszzadamczyk) [inline src/app/shared/constants/theme.ts:67]: here should be shades of red? 🧐

**Review by lukaszzadamczyk — CHANGES_REQUESTED** [review]

File: src/app/shared/constants/theme.ts (hunk 1 above)
Comment (bobbylej) [reply src/app/shared/constants/theme.ts:67]: I've changed them to monochromatic green colors 😉 🟢

**Review by lukaszzadamczyk — APPROVED** [review]

**Review by lukaszzadamczyk — APPROVED** [review]
the last one ? 😁😂

**Comment by bobbylej** [comment]
> the last one ? 😁😂

dunno 🤷

**Pull request merged**